lib/storage/storage_test.go

```diff
@@ -961,9 +961,6 @@ func testStorageRegisterMetricNames(s *Storage) error {
 		return fmt.Errorf("unexpected error in TagFilters.Add: %w", err)
 	}
 	metricNames, err := s.SearchMetricNames(nil, []*TagFilters{tfs}, tr, metricsPerAdd*addsCount*100+100, noDeadline)
-	if err != nil {
-		return fmt.Errorf("error in SearchMetricNames: %w", err)
-	}
```

Comment:
I think it is better to remove the second unused error. Because this one told you about real problem

Reply:
ok